packaging/dev-shell.nix

```diff
@@ -117,7 +117,6 @@ pkgs.nixComponents.nix-util.overrideAttrs (
         pkgs.buildPackages.changelog-d
         modular.pre-commit.settings.package
         (pkgs.writeScriptBin "pre-commit-hooks-install" modular.pre-commit.settings.installationScript)
-        inputs.nixfmt.packages.${pkgs.hostPlatform.system}.default
```

Comment:
This should be needed for the pre-commit hooks which run in the CI, right?

Reply:
We removed this input and are getting it from Nixpkgs, because the flake used IFD.